packages/web3-providers/src/types.ts

```diff
@@ -462,3 +462,12 @@ export namespace MaskBaseAPI {
 
     export interface Provider {}
 }
+
+export namespace ALCHEMY_API {
```

Comment:
The alchemy API should implement `NonFungibleTokenAPI.Provider`. Please remove this.